tests/bpf2c_tests/expected/test_utility_helpers_dll.txt

```diff
@@ -0,0 +1,264 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#define WIN32_LEAN_AND_MEAN // Exclude rarely-used stuff from Windows headers
+// Windows Header Files
+#include <windows.h>
+
+#include <stdio.h>
+
+#include "bpf2c.h"
+
+#define metadata_table test_utility_helpers##_metadata_table
+extern metadata_table_t metadata_table;
+
+BOOL APIENTRY
+DllMain(_In_ HMODULE hModule, unsigned int ul_reason_for_call, _In_ void* lpReserved)
+{
+    UNREFERENCED_PARAMETER(hModule);
+    UNREFERENCED_PARAMETER(lpReserved);
+    switch (ul_reason_for_call) {
+    case DLL_PROCESS_ATTACH:
+    case DLL_THREAD_ATTACH:
+    case DLL_THREAD_DETACH:
+    case DLL_PROCESS_DETACH:
+        break;
+    }
+    return TRUE;
+}
+
+void
+division_by_zero(uint32_t address)
+{
+    fprintf(stderr, "Divide by zero at address %d\n", address);
+}
+
+#define FIND_METADATA_ENTRTY(NAME, X) \
+    if (std::string(NAME) == #X)      \
+        return &X;
+
+metadata_table_t*
+get_metadata_table()
+{
+    return &metadata_table;
+}
+
+// Do not alter this generated file.
+// This file was generated from test_utility_helpers.o
```

Comment:
Seems like this should be at the top of the file, not the middle, or people will think they can modify the code above here and it will be kept if the file is regenerated.